makefile

```diff
@@ -166,9 +223,9 @@ run-tests/%: tests/%.dx just-build
 run-examples/%: examples/%.dx just-build
 	misc/check-quine $< $(dex) script --allow-errors
 
-# Run these with profiling on while they're catching lots of crashes
+# Run these with debugging on while they're catching lots of crashes
 prop-tests: cbits/libdex.so
-	$(STACK) test $(PROF)
+	$(STACK) test $(DBG)
```

Comment:
I think this target can be safely deleted

Reply:
Done.